scripts/Tools/pointclm/pointCLM.py

```diff
@@ -132,36 +132,23 @@
 #                   default=False,action="store_true")
 parser.add_option("--nofire", action="store_true", dest="nofire", default=False, \
                     help="To turn off wildfires")
-parser.add_option("--npoolmod", action="store_true", dest="npoolmod", default=False, \
-                    help="To turn on nitrogen pool modifications")
-parser.add_option("--cpoolmod", action="store_true", dest="cpoolmod", default=False, \
-                    help="To turn on carbon storage pool modifications")
-
-parser.add_option("--q10wbmod", action="store_true", dest="q10wbmod", default=False, \
```

Comment:
the is no longer a pointclm (PTCLM) in the ncar esmci. Since it is a land-component specific utility it is in  components/clm/tools/shared/PTCLM (i.e. within the clm source).  It does not belong in esmci in my opinion.

[3:25] 
just a note from the CESM PTCLM:

[3:25] 
PTCLMmkdata is a python tool built on top of CLM tools and CESM scripts
for building datasets to run CLM "I" cases for data from Ameriflux Tower-sites,
or other user-supplied single-point datasets.